.spr.yml

```diff
@@ -1,6 +1,8 @@
-githubRepoOwner: ejoffe
+githubRepoOwner: wade13
```

Comment:
.spr.yml is repo specific, should not push this update to ejoffe/spr

Reply:
Yeah, this won't be in the final PR, thanks for pointing it out. I was just showing it for the example.